04-Parallel-Simulations-With-MPI/02-Domain-Decomposition.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Wouldn't a simulation box split in this way have splits at y=0 and z=0, since the box goes from -5 to 5 in all directions?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Yes, you are correct.